README.md

```diff
@@ -299,6 +299,52 @@ Our "incoming" object is of type `OneVehicle` but into that component we treat i
 ### Helpers
 
 - `onFormUpdate` hook: Allows you to react whenever the form is being modified. Instead of subscribing to `this.formGroup.valueChanges` or `this.formControls.someProp.valueChanges` you will not have to deal with anything asynchronous nor have to worry about subscriptions and memory leaks. Just implement the method `onFormUpdate(formUpdate: FormUpdate<FormInterface>): void` and if you need to know which property changed do a check like the following: `if (formUpdate.yourProperty) {}`. Be aware that this method will be called only when there are either local changes to the form or changes coming from subforms. If the parent `setValue` or `patchValue` this method won't be triggered
+- `getFormGroupControlOptions` hool: Defines control options for construction of the internal FormGroup. Use this to define form-level validators.
```

Comment:
`hool` typo, should be hook

Reply:
While we're here, I'm being picky but I'd say avoid `.` at the end of a list item

`Use this to define form-level validators.` --> `Use this to define form-level validators`